docs/admin/kubeadm.md

```diff
@@ -370,13 +437,12 @@ parallelize the token distribution for easier automation. To implement this
 automation, you must know the IP address that the master will have after it is
 started.
 
-1.  Generate a token.  This token must have the form  `<6 character string>.<16
-    character string>`.  More formally, it must match the regex
+1.  Generate a token. This token must match the regex
```

Comment:
Many people aren't comfortable with regex.  Having the the "plain english" description will help those folks and make the documentation more approachable.